app/javascript/controllers/navbar_controller.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener('turbo:load', function () {
```

Comment:
at this point this is just vanilla js stored in a folder for stimulus controllers.

why don't you use an actual stimulus controller? :)

Reply:
Great point! I am new to stimulus but I _should_ follow the convention. I will work on it sometime this week. Thanks!

Reply:
hello @awalsh3 thanks for taking this issue. 

I would like to suggest an alternative path at the same time with less JS or maybe none. the details/summary html tags can be used to create dropdowns. This is What Basecamp uses in Hey as an example 

Here is an article on that subject but they are also several others available :

https://dev.to/ayushn21/details-summary-tags-html-s-best-kept-secret-4ak4

Reply:
When I mentioned Basecamp/Hey it was more for the technical part not really the way it looks. Your personal site looks clean I am sure you can make a good judgement on what looks best. I don't have a very strong option on this.